contents/blog/hightouch-posthog-reverse-etl-integration.md

```diff
@@ -0,0 +1,78 @@
+---
+date: 2022-02-24
+title: You can now reverse ETL into PostHog with Hightouch
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Product updates"]
+---
+
+We're delighted to announce that [Hightouch](https://hightouch.io/) has released an integration with PostHog so you can sync modeled data from your data warehouse into PostHog.
+
+## What is Hightouch?
+
+Hightouch is an alternative to Customer Data Platforms (CDP) like Segment, though it can be used in conjunction with them. It allows data teams and engineers to sync data from centralized data warehouses (e.g. Google BigQuery, Redshift, Snowflake etc.) to products like PostHog. This is called Reverse ETL – that's extract, transform and load.
```

Comment:
```suggestion
Hightouch is an alternative to Customer Data Platforms (CDP) like Segment, though it can be used in conjunction with them. It allows data teams and engineers to sync data from centralized data warehouses (e.g. Google BigQuery, Redshift, Snowflake etc.) to products like PostHog. This is called reverse ETL – that's extract, transform and load.
```